build.gradle

```diff
@@ -6,12 +6,15 @@ buildscript {
     }
   }
   dependencies {
+    classpath group: 'com.cinnober.gradle', name: 'semver-git', version: '2.2.0'
     classpath 'com.linkedin:gradle-dustjs-plugin:1.0.0'
     classpath 'de.obqo.gradle:gradle-lesscss-plugin:1.0-1.3.3'
     classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.8'
   }
 }
 
+apply plugin: 'com.cinnober.gradle.semver-git'
+apply plugin: 'idea'
```

Comment:
@juhoautio yes. IntelliJ can understand gradle projects but the support is not that great. Gradle idea plugin is much more robust in building an intelliJ project. Also, you can configure version control, etc so it becomes very easy for a new contributor to download, run "gradle idea", launch the project and start working.

Like the eclipse plugin, idea task is run only when invoked. distTar doesn't have any reason to declare idea as a dependent task. In gradle, the list of tasks to be run is constructed using a dependency task graph. Running **distTar** will invoke **build** since, **distTar** dependsOn **build** and **build** dependsOn **compilaJava** and so on and so forth. Unless a task A is dependent on task B in the task graph, B will not be invoked.